notebooks/cybert/cybert_example_training.ipynb

```diff
@@ -27,6 +27,7 @@
    "outputs": [],
```

Comment:
<p>All of the labels/ field names are printing with the first character missing. And the errors from seqeval.metrics are a result of the missing character.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/327/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
@efajardo-nv Did this get addressed?

Reply:
@BartleyR @raykallen and i are still looking into this.

Reply:
@BartleyR pushed @raykallen's notebook updates. should be good now.